alpine/Dockerfile

```diff
@@ -12,11 +12,17 @@ COPY kong.tar.gz /tmp/kong.tar.gz
 ARG KONG_VERSION=2.2.1
 ENV KONG_VERSION $KONG_VERSION
 
+ARG KONG_AMD64_SHA="e05c7075ae263bb7ef84b0370bf4e31f5246a41d910e5d4336216ae95b654af1"
+ENV KONG_AMD64_SHA $KONG_AMD64_SHA
+
+ARG KONG_ARM64_SHA="c7b2c0e1f47e3009e55b492344bd1ff689a41eca91c21f2c3f57f9172932c96d"
+ENV KONG_ARM64_SHA $KONG_ARM64_SHA
```

Comment:
I think this is unsupported in the official docker build system

Reply:
@tianon Apologies to bug, but could you comment -- is this something unsupported by the Docker Library build system?

Reply:
could be I'm thinking of
> Unfortunately our build systems cannot handle variables in COPY or FROM statements.

trying to locate a past PR where we had thing setup in a similar fashion to see why we didn't go forward with it

Reply:
IMO it's a bit "extra", but it's fine (and is done for `KONG_VERSION` just above this).

Reply:
Thanks for commenting, @tianon!